src/migrations/20180803001052_admin_project.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @param  {object} knex
+ * @return {Promise}
+ */
+export function up(knex) {
+  return knex.schema.createTable('admin_project', table => {
```

Comment:
Be consistent. Use either plural or singular for table names. We usually use plural.